com.microsoft.teams.Teams.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "com.microsoft.teams.Teams",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "19.08",
+  "command": "teams",
+  "tags": [
+    "proprietary"
+  ],
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=all",
+    "--filesystem=home",
```

Comment:
Limit it to downloads please.

Reply:
You can also upload files from your computer. Limiting it to downloads is too restrictive for a good user experience.

Reply:
It's easy to override permissions. We default to security where it's achievable.

Reply:
@barthalion I'm back at it again. [Is doing the same as the slack wrapper acceptable ?](https://github.com/flathub/com.slack.Slack/blob/0650ab1f3cc6c9f5425fa74f8de565fe39aab7b1/com.slack.Slack.json#L16)

Reply:
Yes, that would be better.